glossary.md

```diff
@@ -39,3 +39,21 @@ USA -- typeOf ------------> Country
 USA -- name --------------> United States of America
 USA -- containedInPlace --> northamerica
 ```
+
+- **[Measurement method](https://datacommons.org/browser/measurementMethod)**
+
+The technique used for measuring a statistical variable. Describes how a measurement is made, whether by count or estimate or some other approach. May name the group making the measurement to indicate a certain organizational method of measurement is used. Examples include `BosniaCensus` and [`WorldHealthOrganizationEstimates`](https://datacommons.org/browser/WorldHealthOrganizationEstimates).
+
+- **[Observation period](https://datacommons.org/browser/observationPeriod)**
+
+The time period over which an observation is made.
+
+- **[Unit](https://datacommons.org/browser/unit)**
+
+The unit of measurement. Examples include Kelvin, Celsius, inches, light years, and slugs.
```

Comment:
Do we have light years and slugs? Would be nice to link to the browser for each of these examples.

Reply:
Regretfully, we have neither. I will update and add browser links.